docs/user-guide/building-isos.rst

```diff
@@ -0,0 +1,88 @@
+.. _building-isos:
+
+*************
+Building ISOs
+*************
+
+Since Cobbler uses the systemd hardening option "PrivateTmp" you can't write or read files from your ``/tmp`` when you
+run Cobbler via systemd as a service.
+
+Per default this builds an ISO for all available systems and profiles.
+
+If you want to generate multiple ISOs you need to execute this command multiple times (with different ``--iso`` names).
+
+Under the hood
+##############
+
+Under the hood tool xorriso is used which is being executed in the mkisofs (the predecessor) compatibility mode.
```

Comment:
Are you sure if this sentance is correct. If I understand correctly `xorriso` has a compatibility mode for command line option with `mkisofs`, but is not calling it directly and `mkisofs` also is not calling `xorriso`, but this sentance saying that mkisofs is calling `xorriso`.

Reply:
Should be fixed with d099022